ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Claims.scala

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.v1
+
+import com.digitalasset.daml.lf.data.Ref
+
+/**
+  * [[Claims]] define what actions an authenticated user can perform on the ledger API.
+  *
+  * @param admin True if the user is allowed to use admin services.
+  *
+  * @param party The DAML party that the user is allowed to act as, if any.
+  *              This includes sending commands on behalf of that party and
+  *              reading data for that party.
+  *
+  * The following is a full list of services and the corresponding required claims:
+  * +-------------------------------------+------------------------+
+  * | Ledger API service                  | Access with            |
```

Comment:
Shouldn't some of the services listed here require the claim of acting on behalf of a party?

Reply:
Yes, this table needs to be edited. I have copied it from the proposal document without thinking.

Reply:
That's a bit harsh. It is a draft PR after all 👍

Reply:
Sorry, didn't mean to be harsh, I just wanted to point at this. :slightly_smiling_face:

Reply:
Oh, you mean the "without thinking" part? Shame on you, Robert. You're too harsh on yourself! :smile:

Reply:
Ah, it's more a disclaimer that the whole table is probably wrong, so don't read too much into its content 😄